flink-contrib/flink-storm-examples/pom.xml

```diff
@@ -364,6 +363,18 @@ under the License.
 							</transformers>
 						</configuration>
 					</execution>
+					<execution>
+						<id>shade-flink</id>
```

Comment:
With this we execute the shade-plugin also for other build-profiles, correct? (same for `flink-examples/flink-examples-streaming`) It would be cool if this would only be done when creating the coverage report, so that the normal build-time is unaffected.